ddl/ddl_worker.go

```diff
@@ -797,6 +798,7 @@ func (w *worker) HandleDDLJobTable(d *ddlCtx, job *model.Job) (int64, error) {
 	// reset the SQL digest to make topsql work right.
 	w.sess.GetSessionVars().StmtCtx.ResetSQLDigest(job.Query)
 	err = w.sess.commit()
+	CleanupDDLReorgHandles(job, w.sessPool)
```

Comment:
Why do we need CleanupDDLReorgHandles here, since it would be called in HandleJobDone?

Reply:
Thank you for the suggestion, I removed it here and only rely on the call in HandleJobDone.